cypress/integration/translations.js

```diff
@@ -7,10 +7,10 @@ import {
   createCategory,
   getCategory,
   updateCategoryTranslation
-} from "../../support/api/requests/Category";
-import { deleteCategoriesStartsWith } from "../../support/api/utils/catalog/categoryUtils";
-import filterTests from "../../support/filterTests";
-import { updateTranslationToCategory } from "../../support/pages/translationsPage";
+} from "../support/api/requests/Category";
+import { deleteCategoriesStartsWith } from "../support/api/utils/catalog/categoryUtils";
+import filterTests from "../support/filterTests";
+import { updateTranslationToCategory } from "../support/pages/translationsPage";
 
 filterTests({ definedTags: ["all"], version: "3.0.0" }, () => {
   describe("As an admin I want to manage translations", () => {
```

Comment:
In line 17 `const startsWith = "Translations";` cause that tests finds this word in first row and they fail.
<img width="687" alt="image" src="https://user-images.githubusercontent.com/29308910/152979376-6e3539bb-da35-483b-8499-460c21305e52.png">

Reply:
changed form Translations to TestTranslations